queries/python/highlights.scm

```diff
@@ -346,3 +346,11 @@
               ;; https://docs.python.org/3/library/stdtypes.html
               "bool" "int" "float" "complex" "list" "tuple" "range" "str"
               "bytes" "bytearray" "memoryview" "set" "frozenset" "dict" "type" "object"))
+
+;; Regex from the `re` module
+
+(call
+  function: (attribute
+              object: (identifier) @_re)
+  arguments: (argument_list . (string) @string.regex)
```

Comment:
As only string_content is the injected region, is it reasonable to only have that part be highlighted with `string.regex`, as opposed to also highlighting the delimiters and modifiers?

Reply:
Done.